hudi-client/src/main/java/org/apache/hudi/HoodieWriteClient.java

```diff
@@ -158,6 +167,10 @@ public static SparkConf registerClasses(SparkConf conf) {
 
   /**
    * Upserts a bunch of new records into the Hoodie table, at the supplied commitTime.
+   *
```

Comment:
While you are at it, can you please fix the comment in line 169. 
"Upsert a batch of new records into Hoodie table at the supplied commitTime" 
Similarly fix all comments elsewhere in this file with "batch" instead of "bunch"